samples/sensor/ti_hdc/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: Nucleo STM32L496ZG TI_HDC example
+tests:
+  test_build:
+    build_only: true
+    platform_whitelist: nucleo_l496zg
+    tags: build
+  test_target:
```

Comment:
I'm not an expert on this part but is this section valid?
Are there rules to add a test_target / harness ? @MaureenHelm any idea?

Reply:
I use such sample configuration sections for target testing software via a CI and target test-boards. Of course if they don't fit the project rules then I can remove them.

Reply:
It is removed to simplify this merge request.